### PR TITLE
No keycloak

### PR DESCRIPTION
**Pull request by KaiVolland**
This modifies the shogun-admin to not rely on keycloak.

Authorizaton will be done via a docker/nginx facade with this approach.

It also adds some smaller enhancements to the generic parts of the admin.

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -50,8 +50,10 @@ const App: React.FC = () => {
       setSwaggerDocs(swaggerDoc);
       const appInfo = await AppInfoService.getAppInfo();
       setAppInfo(appInfo);
-      const userInfo = await userService.findOne(appInfo?.userId);
-      setUserInfo(userInfo);
+      if (appInfo.userId) {
```
Comment (annarieger) [inline src/App.tsx:53]: can `appInfo` object be undefined or null here? if so, check for `userId` will fail

File: src/Component/Menu/Navigation/Navigation.tsx [hunk 2]
```diff
@@ -19,18 +19,19 @@ import config from 'shogunApplicationConfig';
 
 interface OwnProps {
   collapsed?: boolean;
+  entityConfigs?: any[];
```
Comment (annarieger) [inline src/Component/Menu/Navigation/Navigation.tsx:22]:
```suggestion
  entityConfigs?: GeneralEntityConfigType<BaseEntity>[];
```

(related imports should be added as well)

File: src/Service/UserService/UserService.ts [hunk 3]
```diff
@@ -2,17 +2,15 @@ import GenericService from '../GenericService/GenericService';
...
 class UserService extends GenericService<User> {
 
   constructor() {
     super(User, config.path.user);
   }
 
-  static logout(): KeycloakPromise<void, void> {
-    return keycloak.logout();
+  static logout(): Promise<string> {
+    return Promise.resolve('TODO');
```
Comment (annarieger) [inline src/Service/UserService/UserService.ts:13]: minor: Is it possible to provide a bit more meaningful message here? :)

**Review by annarieger — APPROVED** [review]

File: src/Page/Portal/Portal.tsx [hunk 4]
```diff
@@ -44,22 +43,17 @@ export const Portal: React.FC<PortalProps> = () => {
...
-        'X-XSRF-TOKEN': CsrfUtil.getCsrfValueFromCookie(),
-        'Authorization': `Bearer ${keycloak.token}`
+        'X-XSRF-TOKEN': CsrfUtil.getCsrfValueFromCookie()
       }
     };
-    const response = await fetch(
-      `${config.appPrefix}${config.path.configBase}/${_toLowerCase(modelName)}.json`,
-      reqOpts
-    );
+    let modelPath = process.env.NODE_ENV === 'development' ?
```
Comment (ahennr) [inline src/Page/Portal/Portal.tsx:53]:
```suggestion
    const modelPath = process.env.NODE_ENV === 'development' ?
```

**Review by ahennr — APPROVED** [review]

**Pull request merged**